doc/en/developer/source/quickstart/checkout.txt

```diff
@@ -8,8 +8,8 @@ Check out the source code from the git repository.::
 To list the available branches.::
 
   % git branch
-     2.14.x
      2.15.x
+     2.16.x
```

Comment:
The PR looks good, no issues with it. This change is good too, but do we want to really refer to actual branches? It would require some documentation pass every time we cut a new stable... can we refer to "abstract" branches instead?
Anyways, just an idea, this change does not make things worse, actually it just makes the doc up to date for the next couple of months.

Reply:
I hate thinking when starting with a new codebase, so seeing real version numbers helps me :)